static/js/components/dashboard/CourseDescription.js

```diff
@@ -19,7 +19,7 @@ import {
 } from '../../constants';
 import { ifValidDate } from '../../util/date';
 
-const edxLinkBase = urljoin(SETTINGS.edx_base_url, 'courses/');
+export const edxLinkBase = urljoin(SETTINGS.edx_base_url, 'courses/');
```

Comment:
since this is needed by more than one file now, can we move this into `constants.js`? anywhere above the test data definitions should be fine, so anywhere above `ELASTICSEARCH_RESPONSE`